docs/aserto-console/directory/graphql-playground.mdx

```diff
@@ -0,0 +1,21 @@
+---
+sidebar_label: GraphQL playground
+title: Using the GraphQL playground
+description: Using the GraphQL playground to query the Directory GraphQL APIa
+---
+
+# GraphQL playground
+
+You can use the GraphQL Playground right from within the console to query the Directory GraphQL APIs. 
+
+### Use the Directory Playground
+
+Use the [Directory Playground](https://console.aserto.com/ui/directory_v2/playground) to craft a new query from scratch using the current logged in user and the selected tenant.
```

Comment:
the url is no longer `directory_v2` it should be https://console.aserto.com/ui/directory/playground